src/main/java/frc/robot/Commands/SettleOnLine.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Subsystems.Drivetrain;
+
+/**
+ * This command moves the {@code Drivetrain} till both its {@code ColorSensorV3} are on the tarmac's line correct color.
+ *
+ * @author Ofri Rosenbaum
+ */
+public class SettleOnLine extends CommandBase {
+
+    public static final double SPEED = 0.7;
+    private Drivetrain drivetrain;
+
+    /**
+     * The alliance's color.
+     */
+    private Color color;
+
+    public SettleOnLine(Color color) {
+        this.color = color;
+        this.drivetrain = Drivetrain.getInstance();
+    }
+
+    @Override
+    public void execute() {
+        double leftSpeed = !isLeftOnColor() ? SPEED : 0;
+        double rightSpeed = !isRightOnColor() ? SPEED : 0;
+        drivetrain.tankDrive(leftSpeed, rightSpeed);
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        drivetrain.stop();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return isLeftOnColor() && isRightOnColor();
+    }
+
+    private boolean isRightOnColor() {
+        return drivetrain.getRightColor().equals(color);
```

Comment:
the Color is too specific, the color will change according to the lights and shadows that the robot's make